service/src/main/java/greencity/service/ubs/ValuesForUserTableServiceImpl.java

```diff
@@ -50,7 +51,8 @@ public PageableDto<UserWithSomeOrderDetailDto> getAllFields(CustomerPage page, S
             users.getPageable().getPageNumber(), users.getTotalPages());
     }
 
-    private UserWithSomeOrderDetailDto mapToDto(User u) {
+    @VisibleForTesting
```

Comment:
why do we need it here?

Reply:
It is used here https://github.com/ita-social-projects/GreenCityUBS/pull/954/files#diff-f331eed926ccba91320b5e7c37a260a220950a74a51c8ba591f629f1ca867c1eR65